src/components/pages/reuse-page.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+
+import PageHeader from '../page-header';
+import Controls from '../controls';
+import Text from '../text';
+import withHeaderName from '../hoc-helpers/withHeaderName';
+
+const ReusePage = ({ match }) => {
```

Comment:
reuse page is a copy-paste of reduce page, which is a copy-paste of recycle
I understand that you just need it to learn routing, but you should get used to write a code as clean as you can

Reply:
Got it.

Reply:
Created a PageTemplate component. Refactored.